inst/include/Rcpp/iostream/Rstreambuf.h

```diff
@@ -3,6 +3,7 @@
 // Rstreambuf.h: Rcpp R/C++ interface class library -- stream buffer
 //
 // Copyright (C) 2011 - 2017  Dirk Eddelbuettel, Romain Francois and Jelmer Ypma
```

Comment:
2011 - 2017 as the next line is 2021.

Reply:
Sorry, what do you mean here?

Reply:
Typo. Meant 2011-2020.

```
// Copyright (C) 2011 - 2020  Dirk Eddelbuettel, Romain Francois and Jelmer Ypma
// Copyright (C) 2021             (same as above plus you)
```

As with the other comments, this PR cannot add you to prior years.  (We can and will of course adjust copyright noticed whenever we forgot someone for prior commits, but that is a different story....)

Reply:
Aaah, okok. I get it now. I've been thinking about this in the wrong way. :-\